be/src/storage/lake/rowset.h

```diff
@@ -114,13 +116,15 @@ class Rowset : public BaseRowset {
     TabletSchemaPtr _tablet_schema;
     TabletMetadataPtr _tablet_metadata;
     std::vector<SegmentSharedPtr> _segments;
+    int64_t _version;
 };
 
 inline std::vector<RowsetPtr> Rowset::get_rowsets(TabletManager* tablet_mgr, const TabletMetadataPtr& tablet_metadata) {
     std::vector<RowsetPtr> rowsets;
     rowsets.reserve(tablet_metadata->rowsets_size());
     for (int i = 0, size = tablet_metadata->rowsets_size(); i < size; ++i) {
-        auto rowset = std::make_shared<Rowset>(tablet_mgr, tablet_metadata, i);
+        auto version = tablet_metadata->rowsets(i).version();
```

Comment:
what if the version is not set in an older version of tablet meta, the version will be 0, will it hurt anything?

I saw a lot of `-1` as not used, `0` will be taken as not used as well?

Reply:
The rowset version is used in cloud native persistent index now. If it is not set, its version will be treated as base_version.

Reply:
if the version is optional for rowset, may better provide a `set_version()` interface to set it when necessary. If not set, the default value will be `-1` or `0` (maybe better 0 as consistent with the one recovered from protobuf).

Reply:
Has removed `_version` from `Rowset`. Version can be returned from `RowsetMetadata`. So the default value of it will be 0.